crud.py

```diff
@@ -0,0 +1,70 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(
+        db: Session,
+        skip: int | None,
+        limit: int | None = None,
+) -> list[models.Author]:
+    query = db.query(models.Author)
+
+    if limit:
+        query = query.offset(limit)
```

Comment:
It seems like you have made a mistake here. You are using the 'limit' variable to offset your query results. This means that you will skip the first 'limit' number of authors. I believe you wanted to limit the number of results instead. You should use the '.limit()' method for this purpose.